src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/enterprise-alert.module.css

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.root {
+	--tag-margin: 12px;
+
+	background: var(--token-color-surface-faint);
+	border: 1px solid var(--token-color-border-faint);
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	padding: 16px;
+	border-radius: 6px;
+	color: var(--token-color-foreground-primary);
+
+	&.inline {
+		background: none;
+		border: none;
+		padding: 0 7px;
+		display: inline-flex;
+		vertical-align: middle;
+
+		--tag-margin: 0;
+
+		&:first-child {
+			padding-left: 0;
+		}
+	}
+
+	@media (--dev-dot-tablet-up) {
+		flex-direction: row;
+	}
+}
+
+.text {
+	/** 
+	* The mdx content anchor styles overwrite the link styles needed 
+	* here, which should inherit body-200 styles
+	*/
+	& > a {
+		font-size: inherit;
+		font-weight: inherit;
+		line-height: inherit;
+	}
+}
+
+.badge {
+	align-self: baseline;
+	margin-right: var(--tag-margin);
+	margin-bottom: var(--tag-margin);
```

Comment:
💭 I _think_ we can replace both of these with `gap` on the parent container, and then the alignment issue Brayden noticed potentially won't be an issue anymore.

Reply:
Nice! i'll refactor to use gap for the base state but the issue brayden pointed out is still a problem due to the padding + inline-flex when the component is set to inline

Reply:
Ah! I see now that was about the headings 💟